modules/web/server/src/main/scala/navigate/web/server/http4s/NavigateMappings.scala

```diff
@@ -462,6 +468,13 @@ class NavigateMappings[F[_]: Sync](
             .map(_.asJson)
             .map(circeCursor(p, env, _))
             .map(Result.success)
+        },
+        RootStream.computeCursor("telescopeState") { (p, env) =>
+          telescopeStateTopic
+            .subscribe(10)
```

Comment:
Since the values are read every second, should duplicate values be filtered out from the subscription? Or do the values change enough with each tick that there will never be duplicates?

Reply:
The same might also be useful for the `guidersQualityValues` subscription

Reply:
In both cases, duplicated values are filtered out before being published in the `Topic`.